packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md

```diff
@@ -168,16 +169,113 @@ You can now run your app by using a local web server. For example, with a macOS,
+
+1. Go to **Applications > Applications** and select your app.
+
+1. Click the **Sign On** tab
+
+1. Scroll down to the **User authentication** section and click **Edit**.
+
+1. From the **Authentication policy** dropdown menu, select your new authentication policy, **Mandatory MFA**. Click **Save**.
+
+1. Test the new configurations by logging into your app. If your test user doesn’t have a phone number enrolled, the user is prompted for the enrollment during the sign in. Any new users signing up also require the enrollment of the phone authenticator.
+
+After your users have enrolled in the phone authenticator, future sign-ins require both a password and SMS code to access your app.
+
+## Enable password recovery with email magic link
+
+By default, the dev org is configured for a self-service password reset. Review the following steps to understand the policy configurations and to enable your sample app users to self-recover their password through an email magic link.
+
+1. Go to **Security > Authenticators** and ensure the email authenticator is available in the **Authenticators** list on the **Setup** tab, and it's used for **Recovery**.
+
+   1. If it isn't listed, click **Add Authenticator**, and then click **Add** in the **Email** tile.
+   {style="list-style-type:lower-alpha"}
+   1. Set **This authenticator can  be used for** to **Recovery**, and click **Add**.
+
+1. Go to **Security > Authenticators** and edit the **Password** authenticator, by clicking **Actions > Edit**.
+
+1. Scroll to the bottom of the **Default Policy**, and edit the **Default Rule**.
+
+1. Ensure that **Password reset** is selected for the **Users can perform self-service** section.
+
+1. Ensure that **Email** is selected for the **Users can initiate recovery with** section.
+
+1. Click **Not required** for **additional verification is** question, and then click **Update rule**.
+
+1. If you previously set the sign-on policy for your app as Mandatory MFA, go to **Applications > Applications** and select your app. Click the **Sign On** tab and scroll down to the **User authentication** section and click **Edit**. From the **Authentication policy** dropdown menu, select **One factor access** and click **Save**.
+
+Test the new configurations by recovering a password for a user of your sample app:
+
+1. Start your app. On the Sign-In Widget, click the **Forgot password?** link.
+
+1. Add the email address for your test user.
+
+1. Click the **Send me an email** link.
+
+1. Check your test user's email inbox and wait for the **Account password reset** email from your Okta dev org.
+
+   >**Note:** Stay in the same browser window as the application sign-in tab.
+
+1. In the body of the email, click the **Reset Password** link.
+
+1. On the **Reset your Okta password** widget, create and verify a new password. Click **Reset Password**. Your test user is now signed in to your sample app with a new password.
```

Comment:
is it a widget? Or is it a dialog? or a page?
```suggestion
1. On the **Reset your Okta password** widget, create and verify a new password. Click **Reset Password**. Your test user is now signed in to your sample app with a new password.
```

Reply:
@susanharper-okta It's a dialog that looks like our widget. It's probably considered part of the SIW, which is why I used the term. But not sure the best way to reference it. Or work around it?